pages/manage_and_operate/iam/iam-logs-forwarding/guide.en-gb.md

```diff
@@ -0,0 +1,215 @@
+
+## Prérequis
+
+- An OVHcloud Account
+- a Logs Data Platform (LDP) account within this OVHcloud Account with at least one active stream configured. [This guide](/pages/manage_and_operate/observability/logs_data_platform/getting_started_quick_start) will walk you through all the necessary steps
+
+## Glossary
+
+**Logs Data Platform**: fully managed and secured log management platform proposed by OVHcloud. Find more information on the [Logs Data Platform service page](https://www.ovhcloud.com/en-ie/logs-data-platform/).
+
+**Data Stream**: a logical partition of logs that you create in an LDP account that you will use when ingesting, visualizing or querying your logs. Multiple sources can be stored in the same data stream, and it is the unit for defining a logs pipeline (retention policy, archiving, live streaming...), access rights and alert policies.
+
+**Logs forwarding**: feature integrated in an OVHcloud product to ingest logs from its services to a data stream of an LDP account in the same OVHcloud account. The feature has to be enabled by the customer per service.
+
+**Logs forwarding Subscription**: when enabling the logs forwarding for a given OVHcloud service to a given LDP data stream, a subscription is created and attached to the data stream for further management by the customer.
+
+**OVH Account audit logs**: Log entries for sensible activities happening within your OVH Account customer main need to collect and use for compliance purposes and other business policy enforcement.
+
+**OVH Account activity logs**: Log entries for all API calls done through OVHCloud API including those triggered when using OVHcloud manager.
+
+**OVH Account policy logs**: Log entries for OVH services access acceptance or denied access due to of a security policy evaluation.
+
+
+## Instruction
+
+### OVHcloud Account logs type
+
+OVHcloud account propose 3 level of logs
+-	**Audit Logs**: provide a security-relevant, chronological set of records documenting the sequence of actions in your OVHCLOUD Account. (ie. logins, password change, …)
+-	**Activity Logs**: provide all of records of actions in your OVHCLOUD Account. (ie. API calls and actions done on the control panel
+-	**Access policy logs**: provide all of records of access evaluation in your OVHCLOUD Account. (ie. Actions authorized or unauthorized through IAM policies)
+
+### Enable logs forwarding
+
+Currently, you can only enable the forwarding of the OVHcloud account logs via API. You will have to define the target Stream of one of your LDP account you aim these logs to be forwarded to. The enablement of the forwarding will create a subscription for this stream id.
+Note that the forwarding enablement is free of charge, but you will be charged for the usage of Logs Data Platform service as per standard price plan. For LDP pricing refer to [Logs Data Platform product page](https://www.ovhcloud.com/en-ie/logs-data-platform/).
```

Comment:
Note that enabling the forwarding is free of charge, but you will be charged for the usage of Logs Data Platform service as per standard price plan. For LDP pricing refer to [Logs Data Platform product page](https://www.ovhcloud.com/en-ie/logs-data-platform/).